### PR TITLE
Slack notifications

### PR DESCRIPTION
**Pull request by tomaj**
Added new recipe for sending messaging to Slack messaging system

### CODE REVIEW

**Comment by gordalina** [comment]
Thanks :smile:

File: recipes/slack.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+/**
+ * Notify Slack of successful deployment
+ */
+task('deploy:slack', function () {
+    global $php_errormsg;
```
Comment (elfet) [inline recipes/slack.php]: What is it?

Comment (tomaj) [inline recipes/slack.php]: What do you mean? Slack task or copyright comment?

Comment (gordalina) [inline recipes/slack.php]: I Think @elfet is referring to php_errormsg, which is used to create the runtime exception.
Also, this recipe along with newrelic can throw an exception, which if thrown will break the deploy, even though it was successful. 
There should be an option to display non fatal errors. 
I proposed https://github.com/deployphp/deployer/pull/108 but it wasn't a good solution.

Comment (tomaj) [inline recipes/slack.php]: Understand. This slack notifier is basically the same as newrelic and have same issues as newrelic. Let me know if there is any better way how to display warning/error from this task and not break whole deployment.

**Comment by tomaj** [comment]
Can you please update Packagist for this repo?

**Comment by gordalina** [comment]
Tagged as 1.0.0

**Pull request merged**